### ISSUE TITLE
.sdkman/tmp not empty

### ISSUE THREAD

**Issue by fouvry** [sdkman/sdkman-cli#575]
I would expect that installation scripts (try to) remove any files from the tmp directory, but the scripts seem to be leaving some stuff behind. Is that intentional and/or can anything be done about it?

**Comment by marc0der** [comment]
You are right, we should add a step to clean up tmp either periodically or after each install.

**Comment by marc0der** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Flush bin files from temp](https://trello.com/c/63x1L6Xu/12-flush-bin-files-from-temp)
